app/models/movie.rb

```diff
@@ -1,31 +1,27 @@
 class Movie < ApplicationRecord
 
-    belongs_to :user
-    has_many :review, dependent: :delete_all
-    has_many :categorizations, dependent: :delete_all
-    has_many :genres, through: :categorizations
+  belongs_to :user
+  has_many :review, dependent: :delete_all
+  has_many :categorizations, dependent: :delete_all
+  has_many :genres, through: :categorizations
 
-    validates :title, presence:true
-    validates :overview, presence:true
-    validates :poster_path, presence:true
-    validates :release_date, presence:true
+  validates :title, presence:true
+  validates :overview, presence:true
+  validates :poster_path, presence:true
+  validates :release_date, presence:true
 
-    accepts_nested_attributes_for :categorizations, :reject_if => :all_blank, :allow_destroy => true
-    accepts_nested_attributes_for :genres
-    # def facebook
-    #  @movies = Facebook.get_object(current_user.token, '/me/movies?fields=name')
-    #end
-    #
-    #
-   def self.search(pattern)
+  accepts_nested_attributes_for :categorizations, :reject_if => :all_blank, :allow_destroy => true
+  accepts_nested_attributes_for :genres
+
+  def self.search(pattern)
     if pattern.blank?  # blank? covers both nil and empty string
       all
     else
-      where('title LIKE ?', "%#{pattern}%")
+      where('lower(title) LIKE lower(?)', "%#{pattern}%")
     end
   end
 
-   def movie_review(user_id, movie_id)
-     return Review.where(user: user_id, movie_id: movie_id )
-   end
+  def movie_review(user_id, movie_id)
+    return Review.where(user: user_id, movie_id: movie_id )
```

Comment:
@sdsunjay this is ruby, no need for return keyword.

Reply:
> @sdsunjay this is ruby, no need for return keyword.

I removed `return`.